pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -165,6 +169,15 @@ func (n *NodeLabeller) run() error {
 	return err
 }
 
+func skipNode(node *v1.Node) bool {
+	if node.Annotations != nil {
```

Comment:
You don't need to check for `nil` on maps for reading: https://play.golang.org/p/vKE5DXEUdb9